setup.py

```diff
@@ -34,7 +34,7 @@
     install_requires=["packaging", "absl-py", "tensorflow_datasets"],
     python_requires=">=3.7",
     extras_require={
-        "tests": ["flake8", "isort", "black", "pytest"],
+        "tests": ["flake8", "isort", "black", "pytest", "tf-models-official"],
```

Comment:
Is this going to change our relationship with models garden or is this just an experiment?

Reply:
This is more for an experiment for the moment.